app/controllers/concerns/finding_aids_finder.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module FindingAidsFinder
+  extend ActiveSupport::Concern
+
+  def has_finding_aids(collection)
+    unless number_of_aids(collection) == 0
```

Comment:
Unless there's good reason to use an unless/else block I'd reduce this to one line of code like `number_of_aides(collection) != 0`. This may be all you need in this method.